docs/guides/integration-mobile-client-server.md

```diff
@@ -0,0 +1,309 @@
+
+If the `getAdvertisingToken()` method call returns `null`, there was no identity or valid token generated. Some possible reasons for this, and some things you could do to troubleshoot, are as follows:
+
+- The identity is invalid. In this scenario there are a couple of options:
+  - Check to see whether there are any errors from the previous `setIdentity()` call.
+  - Check the status of the identity, using one of the following:
+    - **Android Java**: `EUIDManager.getInstance().getCurrentIdentityStatus()` 
+    - **Android Kotlin**: `EUIDManager.getInstance().currentIdentityStatus()` 
+    - **iOS**: `EUIDManager.shared.identityStatus`
+- You could enable logging to get more information: see [Enable Logging](#enable-logging).
+- The advertising token inside the EUID identity has expired, and the refresh token has also expired, so the SDK cannot refresh the token.
+
+If there is no identity, follow the instructions in [Implement Server-Side Token Generation](#implement-server-side-token-generation) again, generate a new identity, and pass the result into your mobile app's EUIDManager again.
+
+## When to Pass a new EUID Identity/Token into the SDK
+
+The best way to determine whether a new EUID identity is required by the EUID SDK again is to call the `getAdvertisingToken()` method in all cases:
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+
+```js
+EUIDManager.getInstance().getAdvertisingToken()
+```
+
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```js
+EUIDManager.shared.getAdvertisingToken()
+```
+
+</TabItem>
+</Tabs>
+
+On startup/resumption of the app, if `getAdvertisingToken()` returns `null`, it is time to generate new identity on the server by following the instructions in [Implement Server-Side Token Generation](#implement-server-side-token-generation). Then, pass the result into the mobile app’s EUIDManager again: see [Configure the EUID mobile SDK](#configure-the-euid-mobile-sdk).
+
+## Enable Logging
+
+<EnableLogging />
+
+## Enable Automatic Token Refresh in Mobile App/Client Side
+
+By default, after a valid EUID identity has been passed into the EUIDManager, it performs automatic token refresh. If for any reason token refresh was disabled, you can enable it with the following method call:
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+
+**Android Java**:
+
+```java
+EUIDManager.getInstance().setAutomaticRefreshEnabled(false)
+```
+
+**Android Kotlin**:
+
+```kotlin
+EUIDManager.getInstance().automaticRefreshEnabled = false
+```
+
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```js
+EUIDManager.shared.automaticRefreshEnabled = false
+```
+
+</TabItem>
+</Tabs>
+
+## Optional: EUID GMA/IMA Plugin for GAM Secure Signals integration
+
+<GMAIMA_Plugins />
+
+## Optional: EUID Prebid Mobile SDK Integration
+
+:::important
+The EUID Prebid Mobile SDK integration is for Android only, and requires version 1.4.0 of the SDK for Android.
```

Comment:
Let's update the version, android sdk version is 1.5.0

Reply:
Done.

Reply:
Done.